docs/agent-reference/how-to-build-agent-code.md

```diff
@@ -70,6 +70,12 @@ downloads but still mocks the `Install` and `Apply` actions:
 ./scripts/build.sh -c -p linux
 ```
 
+To build the reference agent with support for provisioning using the IoT Identity Service or IoTEdge:
+
+```shell
+./scripts/build.sh -c --provision-with-iotedge
```

Comment:
>iotedge [](start = 39, length = 7)

old name from prior to the splitting out of IS from the Edge package, rename?

Reply:
Yep. I'm adding it to the bug we have to change EIS --> IIS/AIS/Whatever.

---
In reply to: [604290362](https://github.com/Azure/iot-hub-device-update/pull/29#discussion_r604290362) [](ancestors = 604290362)